microsoft-edge/web-platform/site-impacting-changes.md

```diff
@@ -15,10 +15,10 @@ The web is constantly evolving to improve the user experience, security, and pri
-| Referrer Policy: Default to `strict-origin-when-cross-origin` | Canary/Dev experimentation starting in M79 | This change is happening in the Chromium project, on which Microsoft Edge is based.  For more information, including the planned timeline by Google for this change, please review the [Chrome Platform Status entry][ChromePlatformStatus6251880185331712].  Based on user and developer feedback, the Microsoft Edge team is shipping this feature in Microsoft Edge at the same time or later than Google.  |  
-| Disallow synchronous XmlHttpRequest in page dismissal | Shipping in M82 | This change is happening in the Chromium project, on which Microsoft Edge is based.  For more information, including the planned timeline by Google for this change, please review the [Chrome Platform Status entry][ChromePlatformStatus4664843055398912].  Based on enterprise and developer feedback, the Microsoft Edge team is shipping this feature in Microsoft Edge later than Chrome.  Matching Chrome, Edge offers a Group Policy to disable this change until Edge 88.  |  
-| Disable TLS/1.0 and TLS/1.1 by default | Shipping in M82 | The `edge://flags/#show-legacy-tls-warnings` flag may be set to cause Edge to show a non-blocking "Not Secure" notice when loading pages that require legacy TLS protocols.  The [SSLMinVersion][DeployedEdgePoliciesSSLMinVersion] Group Policy permits re-enabling of TLS/1.0 and TLS/1.1; the policy remains available until Edge 88.  |  
+| Cookies default to `SameSite=Lax` | Canary/Dev experimentation starting in v82 | This change is happening in the Chromium project, on which Microsoft Edge is based. For more information, including the planned timeline by Google for this change, please review the [Chrome Platform Status entry][ChromePlatformStatus5088147346030592]. Based on user and developer feedback, the Microsoft Edge team is shipping this feature in Microsoft Edge later than Google.  |  
+| Referrer Policy: Default to `strict-origin-when-cross-origin` | Canary/Dev experimentation starting in v79 | This change is happening in the Chromium project, on which Microsoft Edge is based. For more information, including the planned timeline by Google for this change, please review the [Chrome Platform Status entry][ChromePlatformStatus6251880185331712]. Based on user and developer feedback, the Microsoft Edge team is shipping this feature in Microsoft Edge at the same time or later than Google.  |  
+| Disallow synchronous XmlHttpRequest in page dismissal | Shipping in v83 | This change is happening in the Chromium project, on which Microsoft Edge is based. For more information, including the planned timeline by Google for this change, please review the [Chrome Platform Status entry][ChromePlatformStatus4664843055398912]. Based on enterprise and developer feedback, the Microsoft Edge team is shipping this feature in Microsoft Edge later than Chrome. Matching Chrome, Edge offers a Group Policy to disable this change until Edge 88.  |  

```

Comment:
Can we change "Shipping in v83" to "Coming soon"?

Reply:
Can we change the first couple of sentences to align with the feedback in the above cells?

Reply:
Can we change:
Matching Chrome, Edge offers a Group Policy to disable this change until Edge 88.  |
to:
Similar to Chrome, Edge has a Group Policy setting to disable this change up to v88.  |

Reply:
> Can we change "Shipping in v83" to "Coming soon"?

The goal of this document is to communicate a specific timeframe, to the extent possible, to developers and Enterprise IT. The proposed change is in direct contradiction to that goal.

Reply:
Created suggestion with v88 request.

Reply:
Created suggestion with Matching request.

Reply:
Moved the conversation to email.